crates/provider-http-server/src/address.rs

```diff
@@ -287,8 +289,10 @@ impl HttpServerCore {
         } else {
             debug!(?addr, "bind HTTP listener");
 
+            let mut srv = axum_server::from_tcp(listener);
+            srv.http_builder().http1().keep_alive(false);
```

Comment:
This is the only part i'm not sure if we should do.

The behaviour of a localhost / non-tls scenario test vs networked + tls will be quite different.

localhost vs networked
- For linux, `tcp_tw_reuse` and `tcp_fin_timeout` act differently, basically won't run out of ports.
- When networked, each of those connections will need roundtrips for syn/syn-ack & fin/fin-ack

plain vs tls
- tls needs 2 roundtrips to establish session ( hello + key exchange )

I'd either leave this one out or validate the performance is consistent in a networked environment

Reply:
I haven't tested TLS indeed, I'll remove it for now